### PR TITLE
ci(travis): add lint and fmt check

### PR DESCRIPTION
**Pull request by jhyeom26**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,19 +6,31 @@ before_cache:
...
 env:
   global:
     - PATH="$PATH:$HOME/protoc/bin"
     - RUST_BACKTRACE=1
 install:
-  - ./scripts/install_protoc.sh
-script:
   - mkdir -p integration-testing/resources
+  - ./scripts/install_protoc.sh
+  - nvm install 10
```
Comment (nowooj) [inline .travis.yml]: How about using `lts` if there is no specific version dependency?

```
$ nvm install --lts
$ nvm use stable
```

Comment (jhyeom26) [reply .travis.yml]: :+1: applied in 97d089b

**Comment by psy2848048** [comment]
스크린샷 2020-02-20 오후 2 31 10

I don't know how to control in YAML, but is there any option to turn off Branch build?

**Comment by jhyeom26** [comment]
> 스크린샷 2020-02-20 오후 2 31 10
> 
> I don't know how to control in YAML, but is there any option to turn off Branch build?

Yap, this can be done in [here](https://travis-ci.com/hdac-io/CasperLabs/settings).
I've applied that

**Review by nowooj — APPROVED** [review]
Doing forec-push on the features branch ...

**Comment by jhyeom26** [comment]
> Doing forec-push on the features branch ...

:ok_hand:  I will follow `private` -> `features` -> `master` way next time.

But I hope to discuss this strategy.
Most of the features can be done independently so they are not always to be treated as a shared branch. I feel this is quite a workaround to create `private/name` branch for every feature branch.
Although we will not share this branch, we have to create `private/name` and need to get double reviews`(private -> feature -> master)`.
Moreover, we cannot guess what `private/name` implements by its name.

So I suggest instead of using `private/name`, using `features/content` for all features and if some feature is implemented by the several people, then, we can prevent that branch to be force-pushed in the github. In my humble opinion, this is less case than independently developed cases.

I am curious about thoughts on this you guys.
@hdac-io/developers

**Review by psy2848048 — APPROVED** [review]

**Pull request merged**